src/main/java/com/elasticbox/jenkins/triggers/github/PullRequestBuildHandler.java

```diff
@@ -433,19 +434,23 @@ private boolean isWhitelisted(GHUser user, GitHub gitHub) throws IOException {
         return false;
     }
 
-    private boolean isPullRequestBuild(Actionable actionable, String pullRequestUrl) {
+    private boolean isPullRequestBuildInProject(Actionable actionable, String pullRequestUrl, String projectName) {
+        String itemProjectName = null;
+        String itemRequestUrl = null;
         ParametersAction parameters = actionable.getAction(ParametersAction.class);
 
         if (parameters == null) {
             return false;
         }
         for (ParameterValue parameter: parameters.getParameters()) {
-            if ("PR_URL".equals(parameter.getName()) && pullRequestUrl.equals(parameter.getValue())) {
-                return true;
+            if (PR_URL.equals(parameter.getName())) {
+                itemRequestUrl = (String) parameter.getValue();
+            } else if (PROJECT_NAME.equals(parameter.getName())) {
+                itemProjectName = (String) parameter.getValue();
             }
         }
 
-        return false;
+        return ( (itemProjectName.equals(projectName)) && (itemRequestUrl.equals(pullRequestUrl)) ) ;
```

Comment:
Please, flip the objects in the equals, to better ensure they are not null

Reply:
Hey @julio-castanar, let's fix this in the next one, I want to test the release job today